.github/workflows/trustee-attester.yml

```diff
@@ -0,0 +1,99 @@
+name: trustee-attester basic build
+
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - 'attestation-agent/**'
+      - '.github/workflows/trustee-attester.yml'
+      - 'Cargo.toml'
+      - 'Cargo.lock'
+  pull_request:
+    paths:
+      - 'attestation-agent/**'
+      - '.github/workflows/trustee-attester.yml'
+      - 'Cargo.toml'
+      - 'Cargo.lock'
+  create:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  trustee_attester_ci:
+    if: github.event_name != 'push'
+    name: Check
+    defaults:
+      run:
+        working-directory: ./attestation-agent/kbs_protocol/
+    strategy:
+      fail-fast: false
+      matrix:
+        rust:
+          - stable
+        instance:
+          - ubuntu-24.04
+        include:
+          - instance: ubuntu-24.04
+            cargo_test_opts: "-p kbs_protocol --bin trustee-attester --no-default-features --features background_check,passport,openssl,all-attesters,bin"
```

Comment:
i don't think this is a matrix. we only run the suite on ubuntu 24.04 w/ rust stable

Reply:
I removed the s390x entry of the list, for now, so only ubuntu is left there.
Possibly someone will re-add it later.

Reply:
I see, maybe add this as a comment above the matrix statement. otherwise, it's unclear to the reader of that workflow.

Reply:
I re-added s390x. It was added to many other .yml files